### ISSUE TITLE
Staging Review finding: Beneficiary travel agreement displays validation error on page load

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#104090]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Modernized Check-in
**Product:** Appointments
**Feature:** Travel Claims for Past Appointments

## Findings details
**VA.gov Experience Standard - issue:** [User experiences predictable interactions with components and patterns.](https://design.va.gov/about/experience-standards/#user-can-identify-an-element-and-its-state)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @erinrwhite (IA), @briandeconinck (Accessibility)

### Description

On the beneficiary travel agreement at the end of the form, the error message `You must accept the beneficiary travel agreement before continuing.` is displaying automatically.

This also adds some extra confusion for screen reader users, who will hear the error message out of context. Error messages on VA.gov have `role="alert"`, which means when they're present on the page they prompt screen readers to announce the message before they announce anything else on the page. For a screen reader user moving through the flow, that means you'll hear "You must accept the beneficiary travel agreement before continuing" before you even know that there's a beneficiary travel agreement and before you've had a chance to review your submission.

<details><summary>Link, screenshot or steps to recreate</summary>

![Image](https://github.com/user-attachments/assets/d9e4ecac-c4f1-473d-a7df-75855e33eec2)

</details>

## Recommended action

Don't display the error message until it's needed.

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** [Link component](https://design.va.gov/components/link)
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** Navigation

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
